aztec/src/main/kotlin/org/wordpress/aztec/formatting/LineBlockFormatter.kt

```diff
@@ -273,32 +270,31 @@ class LineBlockFormatter(editor: AztecText, val headerStyle: LineBlockFormatter.
 
     fun insertMedia(drawable: Drawable?, attributes: Attributes, onMediaTappedListener: OnMediaTappedListener?) {
         val span = AztecMediaSpan(editor.context, drawable, attributes, onMediaTappedListener)
-        val html = span.getHtml();
 
-        val isAddingMediaToEndOfBlockElement = editableText.getSpans(selectionStart, selectionEnd, AztecBlockSpan::class.java)
-                .any {
-                    selectionStart == editableText.getSpanEnd(it)
-                }
+        val spanAtEndOfWhichMediaIsAdded = editableText.getSpans(selectionStart, selectionEnd, AztecBlockSpan::class.java)
```

Comment:
It looks like `spanAtEndOfWhichMediaIsAdded` was renamed similar to `spanAtStartOfWhichMediaIsAdded`.  What about naming them `spanBeforeMedia` and `spanAfterMedia`, respectively?  That's easier to grok.